contract.md

```diff
@@ -4,6 +4,10 @@
 
 _At a high level, what do you most want to get out of this project, and why?_
 
+I want to design a language that makes sense for non-programmers to explore a DSL as a potential way of guiding people who may feel intimidated by programming to do things they are familiar with programmatically. The language can also be a foray to help people get into programming (similar to the way HTML/CSS are a sort of "jump pad" for learning coding for many people). I am also intrigued by various text processing tools I see on the internet (ex. resume parsing, pdf to word doc, etc.) and would like to explore the domain more through this project.
+
```

Comment:
I like how you are creating your language with those who may feel intimidated by programming in mind. I know many people who feel this way and even felt this way myself at one point; afraid to do anything outside of my comfort zone on the computer in fear that I would end up messing something up. Having a language that doesn't feel like programing is a great way to fix this issue and will ultimately make your DSL suited for a wider audience